src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -109,6 +109,11 @@ export function isPythonNotebook(metadata?: nbformat.INotebookMetadata) {
     if (metadata?.language_info?.name && metadata.language_info.name !== PYTHON_LANGUAGE) {
         return false;
     }
+
+    if (kernelSpec?.name.includes(PYTHON_LANGUAGE)) {
+        return true;
```

Comment:
Could we reuse this existing function? 
https://github.com/microsoft/vscode-jupyter/blob/f274dc1dd7f795094fca6021532cce817d380f6d/src/client/datascience/notebook/helpers/helpers.ts#L106-L114